Makefile

```diff
@@ -101,3 +101,12 @@ mocks: $(MOCKGEN)
 	$(MOCKGEN) --build_flags=--mod=mod -package mocks -destination=cmd/create/idp/mocks/identityprovider.go -source=cmd/create/idp/cmd.go IdentityProvider
 	$(MOCKGEN) --build_flags=--mod=mod -package mocks -destination=pkg/aws/mocks/s3api.go github.com/aws/aws-sdk-go/service/s3/s3iface S3API
 	$(MOCKGEN) --build_flags=--mod=mod -package mocks -destination=pkg/aws/mocks/secretsmanagerapi.go github.com/aws/aws-sdk-go/service/secretsmanager/secretsmanageriface SecretsManagerAPI
+
+.PHONY: e2e_test
+e2e_test: tools install
```

Comment:
I didn't see the tools make CMD in the make file

Reply:
gonna check this

Reply:
updated